pyproject.toml

```diff
@@ -9,18 +9,26 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
+dice_ml = "0.*"
+lightgbm = "3.3.*"
 numpy = "1.24.*"
+optuna = "3.3.*"
 pandas = "1.4.*"
-python = "3.10"
-scikit-learn = "1.2.*"
 pydantic = "1.10.*"
+python = "3.10.*"
```

Comment:
Sam Python jest na tyle ważny, że jego polecam trzymać nad libkami, nawet z linijką odstępu dla zaznaczenia. Ale to kosmetyka